files/inspector.key

```diff
@@ -0,0 +1,29 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Is this key available on any key servers? Might be better go grab it from somewhere like keyserver.ubuntu.com rather than checking it in.

Reply:
That would be much nicer, but I couldn't find one.  [The doco states](https://docs.aws.amazon.com/inspector/latest/userguide/inspector_verify-sig-agent-download-linux.html#inspector_verify-signature-of-agent-download-authenticate-import-public-key):

> 1. Obtain a copy of our public GPG build key by doing one of the following:
>
>   *  Download from https://d1wk0tztpsntt1.cloudfront.net/linux/latest/inspector.gpg.
>
>  *   Copy the key from the following text and paste it into a file called inspector.key. Be sure to include everything that follows:

The download location is the same as the installer script itself, so I copied the key from the website.

Reply:
OK no probs at all.

Reply:
You *could* scrape the key off the website...

```bash
curl -s https://docs.aws.amazon.com/inspector/latest/userguide/inspector_verify-sig-agent-download-linux.html | awk '/-----BEGIN PGP PUBLIC KEY BLOCK-----/ { print "-----BEGIN PGP PUBLIC KEY BLOCK-----"; key=1; next } /-----END PGP PUBLIC KEY BLOCK-----/ { print; key=0 } key'
```

But let's not.